src/main/java/teammates/ui/output/FeedbackSessionData.java

```diff
@@ -18,12 +23,16 @@ public class FeedbackSessionData extends ApiOutput {
     private final String courseId;
     private final String timeZone;
     private final String feedbackSessionName;
+    private String feedbackSessionUserEmail;
     private final String instructions;
 
+    // TODO: set as final after migration, cant do so now since the attributes constructor does not have this.
+    private FeedbackSession feedbackSession;
```

Comment:
This is used for API output, should only be JSON types.

Reply:
Removed